cli/tsc/dts/lib.deno_net.d.ts

```diff
@@ -456,6 +456,12 @@ declare namespace Deno {
      * TLS handshake.
      */
     alpnProtocols?: string[];
+    /** If true, the certificate's common name or subject alternative names will not be
+     * checked against the hostname provided in the options.
+     *
+     * @default {false}
+     */
+    disableHostnameVerification?: boolean;
```

Comment:
Should we name it something like `unsafelyDisableHostnameVerification`? I think we should update the docstring too, to tell that this shouldn't be done by most users and when _it might_ be useful.

Reply:
I was thinking this too. Btw, @littledivy remember to post these public api changes to the cli-public-api channel

Reply:
Renamed to`unsafelyDisableHostnameVerification`